### PR TITLE
Fix broken email fixtures

### PR DESCRIPTION
**Pull request by k127**
Please answer the following questions:

| Q             | A
| ------------- | ---
| Bug fix?      | Yes
| New feature?  | -
| BC breaks?    | -
| Deprecations? | -
| Fixed issues  |  Not to my knowledge

## Description

`./app/console doctrine:fixtures:load`
Careful, database will be purged. Do you want to continue y/N ?y
  > purging database
  > loading [1] Mautic\UserBundle\DataFixtures\ORM\LoadRoleData
  > loading [2] Mautic\UserBundle\DataFixtures\ORM\LoadUserData
  > loading [4] Mautic\LeadBundle\DataFixtures\ORM\LoadLeadFieldData
  > loading [5] Mautic\ReportBundle\DataFixtures\ORM\LoadReportData
  > loading [5] Mautic\LeadBundle\DataFixtures\ORM\LoadLeadListData
  > loading [5] Mautic\LeadBundle\DataFixtures\ORM\LoadLeadData
  > loading [6] Mautic\PageBundle\DataFixtures\ORM\LoadPageCategoryData
  > loading [7] Mautic\PageBundle\DataFixtures\ORM\LoadPageData
  > loading [8] Mautic\FormBundle\DataFixtures\ORM\LoadFormData
  > loading [8] Mautic\PageBundle\DataFixtures\ORM\LoadPageHitData
  > loading [9] Mautic\EmailBundle\DataFixtures\ORM\LoadEmailData

  [Doctrine\DBAL\DBALException]                                                                                                                                                                                                   
  An exception occurred while executing 'INSERT INTO emails (is_published, date_added, created_by, created_by_user, date_modified, modified_by, modified_by_user, checked_out, checked_out_by, checked_out_by_user, name, descri  
  ption, subject, from_address, from_name, reply_to_address, bcc_address, template, lang, content, plain_text, custom_html, email_type, publish_up, publish_down, read_count, sent_count, revision, variant_settings, variant_st  
  art_date, variant_sent_count, variant_read_count, category_id, variant_parent_id, unsubscribeform_id) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with p  
  arams [1, "2016-05-04 16:11:35", null, null, null, null, null, null, null, null, null, null, "Join us at the 2014  Kaleidoscope Conference!", null, null, null, null, "Mauve", "en", "a:2:{s:4:\"body\";s:110:\"<div>Dear {lea  
  dfield=firstname},<\/div><div>&nbsp;<\/div><div>Join us at the 2014 Kaleidoscope Conference!<\/div>\";s:6:\"footer\";s:71:\"<div>{webview_text}<\/div><div>&nbsp;<\/div><div>{unsubscribe_text}<\/div>\";}", "Join us at the 2  
  014  Kaleidoscope Conference!", null, null, null, null, "0", "0", "1", "a:0:{}", null, "0", "0", null, null, null]:                                                                                                             
  SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'name' cannot be null
                                                                                      
  [PDOException]                                                                      
  SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'name' cannot be null  
                                                                                      
doctrine:fixtures:load [--fixtures[="..."]] [--append] [--em="..."] [--shard="..."] [--purge-with-truncate] [--multiple-transactions]

## Steps to reproduce the bug (if applicable)

    ./app/console doctrine:schema:drop --force
    ./app/console doctrine:schema:create
    ./app/console doctrine:fixtures:load

## Steps to test this PR

    ./app/console doctrine:schema:drop --force
    ./app/console doctrine:schema:create
    ./app/console doctrine:fixtures:load

### CODE REVIEW

**Comment by alanhartless** [comment]
Hi @k127! Thanks for submitting this!

There seems to be a lot of commits associated with this PR/branch that shouldn't be. I'm concerned that merging this may cause some regressions if the code from these extra commits have been changed since they were first merged.

Can you see about cleaning this up a bit? It may need a fresh branch/PR or something.

Thanks!
Alan

**Comment by k127** [comment]
Hi Alan, thanks for your response! I'll just try to patch those two relevant changesets onto the current staging, and file a PR, ok?

Cheers
Klaus

**Comment by k127** [comment]
Hi @alanhartless, PR is updated. Cheers, Klaus

File: app/bundles/EmailBundle/DataFixtures/ORM/fakeemaildata.csv [hunk 1]
```diff
@@ -1,2 +1,3 @@
...
+"name","emailType","subject","template","language","content","plainText","publishUp","publishDown","readCount","sentCount","revision","variantSettings","variantStartDate","variantSentCount","variantReadCount"
+"Kaleidoscope Conf List","list","Join us at the 2014  Kaleidoscope Conference!","Mauve","en","a:2:{s:4:\"body\";s:110:\"<div>Dear {leadfield=firstname},</div><div>&nbsp;</div><div>Join us at the 2014 Kaleidoscope Conference!</div>\";s:6:\"footer\";s:71:\"<div>{webview_text}</div><div>&nbsp;</div><div>{unsubscribe_text}</div>\";}","Join us at the 2014  Kaleidoscope Conference!",NULL,NULL,0,0,1,"a:0:{}",NULL,0,0
+"Kaleidoscope Conf Campaign","template","Join us at the 2014  Kaleidoscope Conference!","Mauve","en","a:2:{s:4:\"body\";s:110:\"<div>Dear {leadfield=firstname},</div><div>&nbsp;</div><div>Join us at the 2014 Kaleidoscope Conference!</div>\";s:6:\"footer\";s:71:\"<div>{webview_text}</div><div>&nbsp;</div><div>{unsubscribe_text}</div>\";}","Join us at the 2014  Kaleidoscope Conference!",NULL,NULL,0,0,1,"a:0:{}",NULL,0,0
```
Comment (dongilbert) [inline app/bundles/EmailBundle/DataFixtures/ORM/fakeemaildata.csv]: Please update instances of `{leadfield=` to `{contactfield=` instead, and we can get this merged. Thanks

**Pull request merged**